test/ts/index.ts

```diff
@@ -11,3 +11,19 @@ console.log(
     }
   })
 )
+
+function wrapGetopts(input: string, options: Options): ParsedOptions {
```

Comment:
@Ovyerus If I understand this correctly, you've created `wrapGetopts` to demonstrate how to use `ParsedOptions` and `Options`. 

If that's right,  can we find a different way to do that? In practice, I wouldn't wrap the `getopts` function, so to me, if I was looking at this file for the first time, this function would seem strange and I'd have a hard time trying to understand its purpose—at least for a while.

Reply:
Fair enough. Any suggestions on an alternative?

Reply:
@Ovyerus What about something like this?

```ts
import getopts, { ParsedOptions, Options } from "getopts"

const options: Options = {
  alias: {
    s: "super"
  },
  boolean: ["super"],
  default: {
    super: true
  }
}

const parsedOptions = getopts(["-s"], options)

if (!parsedOptions.super) {
  process.exit(1)
}
```

Reply:
Seems good.